diff --git a/node_modules/react-native-emoji-selector/index.js b/node_modules/react-native-emoji-selector/index.js
index 0737acd..4fca06c 100644
--- a/node_modules/react-native-emoji-selector/index.js
+++ b/node_modules/react-native-emoji-selector/index.js
@@ -91,7 +91,9 @@ const TabBar = ({ theme, activeCategory, onPress, width }) => {
             style={{
               textAlign: "center",
               paddingBottom: 8,
-              fontSize: tabSize - 24
+              // fontSize: tabSize - 24
+              // fontSize: Platform.OS === "android" ? 20 : tabSize - 20
+              fontSize: 20
             }}
           >
             {category.symbol}
@@ -112,7 +114,11 @@ const EmojiCell = ({ emoji, colSize, ...other }) => (
     }}
     {...other}
   >
-    <Text style={{ color: "#FFFFFF", fontSize: colSize - 12 }}>
+    <Text style={{
+      color: "#FFFFFF",
+      // fontSize: colSize - 12
+      fontSize: 20
+    }}>
       {charFromEmojiObject(emoji)}
     </Text>
   </TouchableOpacity>
